src/QubeWireClient.h

```diff
@@ -0,0 +1,124 @@
+/**
+ * @file QubeWireClient.h
+ * @author  Qube Dev <QubeDev@realimage.com>
```

Comment:
I saw this line and thought its fine, but after seeing your comment it looks like its not fine to have an email address